teapots/choreographer-30fps/src/main/cpp/CMakeLists.txt

```diff
@@ -17,36 +17,53 @@
 cmake_minimum_required(VERSION 3.4.1)
 
 # build native_app_glue as a static lib
-add_library(native_app_glue STATIC
-    ${ANDROID_NDK}/sources/android/native_app_glue/android_native_app_glue.c)
-
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11 -Wall -fno-exceptions -fno-rtti")
+add_library(NativeAppGlue STATIC
+  ${ANDROID_NDK}/sources/android/native_app_glue/android_native_app_glue.c)
```

Comment:
Would appreciate keeping the sweeping stylistic changes out of this change since it just makes it harder to review by bloating the diff.